src/cargo/sources/path.rs

```diff
@@ -411,6 +470,228 @@ impl<'gctx> PathSource<'gctx> {
         }
     }
 
+    /// Lists files relevant to building this package inside this source by
+    /// traversing the git working tree, while avoiding ignored files.
+    ///
+    /// This looks into Git submodules as well, resolving them to individual files.
+    /// Symlinks to directories will also be resolved, which may include `.git` folders
+    /// in entirety if the link points to a submodule.
+    fn list_files_gix(
+        &self,
+        pkg: &Package,
+        repo: &gix::Repository,
+        filter: &dyn Fn(&Path, bool) -> bool,
+    ) -> CargoResult<Vec<PathBuf>> {
+        use gix::dir::entry::Status;
+        warn!("list_files_gix {}", pkg.package_id());
```

Comment:
Why is this a `warn`?  Should we instead `instrument`?

Reply:
It's a line that was copied from the `git2` version of the method. I am happy to change it in both once you confirm.

Reply:
Huh, thought I checked for it but must have missed it.  No need to "innovate" here.